crates/kitsune_p2p/kitsune_p2p/src/spawn/actor.rs

```diff
@@ -23,6 +23,15 @@ use std::collections::hash_map::Entry;
 use std::collections::HashMap;
 use std::sync::Arc;
 
+/// Default webrtc config if set to `None`.
+/// TODO - set this to holochain stun servers once they exist!
+const DEFAULT_WEBRTC_CONFIG: &str = r#"{
+  "iceServers": [
+    { "urls": "stun:stun.l.google.com:19302" },
+    { "urls": "stun:stun2.l.google.com:19302" }
+  ]
+}"#;
+
```

Comment:
TODO - update this to match the holochain production stun servers to be created.

Reply:
done